libs/sidebar/use-cases/src/lib/ports/data/sidebar-feature.state.ts

```diff
@@ -2,7 +2,7 @@ import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 
 @Injectable()
-export abstract class SidebarDataPort {
+export abstract class SidebarFeatureState {
```

Comment:
Is there any chance to add also the word `Interface` to the current naming? 😅 I know this is just one of the possible approaches but in this particular case, it would help me to realize pretty fast it's just an abstract class right from the filename.

Reply:
I named it `SidebarFeatureState` on purpose since this is the thing you are using in your components. So when u use it you have it like:
```ts
@Component()
export class MyComponent {
  constructor(private state: SidebarFeatureState) {
  }
}
```